src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -1139,7 +1139,47 @@ public async Task TestQueryFilterFieldAuth_ExcludeTakesPredecence()
         }
 
         /// <summary>
-        /// Tests that the pk level query filter is working variables
+        /// Tests that the field level query filter work with list type for 'contains' operator
+        /// </summary>
+        [TestMethod]
+        public async Task TestQueryFilterContains_WithStringArray()
+        {
+            string gqlQuery = @"{
+                planets(" + QueryBuilder.FILTER_FIELD_NAME + @" : {tags: { contains : ""tag1""}})
+                {
+                    items {
+                        id
+                        name
+                    }
+                }
+            }";
+
+            string dbQuery = $"SELECT c.id, c.name FROM c where ARRAY_CONTAINS(c.tags, 'tag1')";
+            await ExecuteAndValidateResult(_graphQLQueryName, gqlQuery, dbQuery);
+        }
+
+        /// <summary>
+        /// Tests that the field level query filter work with list type for 'notcontains' operator.
+        /// </summary>
+        [TestMethod]
+        public async Task TestQueryFilterNotContains_WithStringArray()
+        {
+            string gqlQuery = @"{
+                planets(" + QueryBuilder.FILTER_FIELD_NAME + @" : {tags: { notContains : ""tag3""}})
+                {
+                    items {
+                        id
+                        name
+                    }
+                }
+            }";
+
+            string dbQuery = $"SELECT c.id, c.name FROM c where NOT ARRAY_CONTAINS(c.tags, 'tag3')";
```

Comment:
Can you please update the PR description that Integration tests have been added - Mark the check box to yes.

Also, provide a sample GQL query and corresponding Cosmos DB query that will be generated.